pkg/diagnostics/http_monitoring_test.go

```diff
@@ -107,6 +107,14 @@ func TestConvertPathToMethodName(t *testing.T) {
 		{"actors/DemoActor/1/method/method1", "actors/DemoActor/{id}/method/method1"},
 		{"actors/DemoActor/1/method/timer/timer1", "actors/DemoActor/{id}/method/timer/timer1"},
 		{"actors/DemoActor/1/method/remind/reminder1", "actors/DemoActor/{id}/method/remind/reminder1"},
+		{"/v1.0-alpha1/workflows/dapr/mywf/start?instanceID=1234", "/v1.0-alpha1/workflows/dapr/mywf/start"},
```

Comment:
Please use special name as `workflowComponentName`, the `dapr` here may mislead.